addons/skin.estuary/xml/SkinSettings.xml

```diff
@@ -82,7 +82,8 @@
 					<label2>$INFO[Skin.String(background_overlay),$LOCALIZE[467] ]</label2>
 					<include>DefaultSettingButton</include>
 					<onclick condition="String.IsEqual(Skin.String(background_overlay),1)">Skin.SetString(background_overlay,2)</onclick>
-					<onclick condition="String.IsEqual(Skin.String(background_overlay),6)">Skin.SetString(background_overlay,1)</onclick>
+					<onclick condition="String.IsEqual(Skin.String(background_overlay),7)">Skin.SetString(background_overlay,1)</onclick>
```

Comment:
this one is listed twice:
<onclick condition="String.IsEqual(Skin.String(background_overlay),1)">Skin.SetString(background_overlay,2)</onclick>